lib/src/utf16.dart

```diff
@@ -8,15 +8,15 @@ import 'dart:typed_data';
 
 import 'package:ffi/ffi.dart';
 
-/// [Utf16] implements conversion between Dart strings and null-terminated
+/// [Utf16] implements conversion between Dart strings and zero byte terminated
```

Comment:
Nit: It's not a zero *byte* when it's 16-bit. Maybe a a zero "word" or "code unit". Maybe just "zero terminated", then it's the zero value of the appropriate type.